src/components/SideInformationsContainer/ExplanationCard/ExplanationCard.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import * as Styles from "./styles";
+
+export default function ExplanationCard(){
+
+    const [cardStatus, setCardStatus] = useState(false);
+
+    return(
+        <Styles.ExplanationContainer isCardClosed={ cardStatus } >
+            <Styles.ExplanationCardCloseIcon onClick={ ()=>{ setCardStatus(true); }  }  width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/Styles.ExplanationCardCloseIcon" 
+            >
+                <path fill-rule="evenodd" clip-rule="evenodd" d="M0.439341 9.43589C-0.146446 10.0217 -0.146447 10.9714 0.439339 11.5572C1.02513 12.143 1.97487 
```

Comment:
Try to use the material ui component, there we have the [Close icon](https://mui.com/material-ui/material-icons/?query=close&selected=Close), [Here](https://github.com/gpmc-lab-ufrgs/atlas/blob/develop/src/components/Header/styles.ts#:~:text=import%20%7B%20Menu%20as%20MenuIcon%20%7D%20from%20%22%40mui/icons%2Dmaterial%22%3B) a example in our code using this lib

Reply:
Greate advice, thanks.